packages/core/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -258,9 +258,10 @@ export const Actions: StoryObj<HvAvatarProps> = {
           overrideIconColors={false}
           component="a"
           href="#profile-url"
+          aria-label="External link"
         >
           <HvAvatar size="md">
-            <LinkIcon color="base_dark" aria-label="External link" />
+            <LinkIcon color="base_light" />
```

Comment:
This way it doesn't work in wicked 😅 This needs to be `atmo1`, like the default Avatar icon

![image](https://github.com/lumada-design/hv-uikit-react/assets/638946/69837cf1-5e00-4812-b445-6fb630260053)


```suggestion
            <LinkIcon color="atmo1" />
```

Reply:
Updated!